### ISSUE TITLE
Use doctr

### ISSUE THREAD

**Issue by certik** [sympy/planet-sympy#64]
Using `doctr` might simplify quite a few lines in our deploy script:

https://github.com/drdoctr/doctr

@asmeurer mentioned that perhaps https://github.com/drdoctr/doctr/issues/189 must be fixed first, though perhaps not, since we actually run this inside Docker, whether the docker container is run on Travis-CI or GitLab-CI makes no difference to the code within.

**Comment by asmeurer** [comment]
It does matter. The point of doctr is to manage the deploy keys. Travis CI has a [special](https://docs.travis-ci.com/user/encryption-keys/) public/private key pair for each repository, which we use to encrypt an SSH deploy key for pushing to GitHub pages. This is 90% of what doctr does, and the main reason that I wrote it, because it's such a pain to do this manually all the time (the other 90% is managing the syncing of files).

GitLab hopefully has something similar, or (better), a way to upload SSH keys directly.

**Comment by certik** [comment]
@asmeurer the way I've done it, for both Travis and GitLab-CI is to literally paste the (base64 encoded) private ssh key as a private variable (at either Travis or GitLab) and it then gets passed into the script as a variable. So it's simple, and as secure as your solution.

**Comment by asmeurer** [comment]
Yeah. The point of doctr is to do that automatically, using the API. So you don't have to go in and paste stuff in websites. You just run one command (`doctr configure`), enter your passwords, and it's all done automatically. Then the only thing you have to do manually is add the encrypted key to the .travis.yml (for GitLab, nothing would have to be done manually).
